src/main/java/org/folio/linked/data/integration/kafka/sender/search/KafkaSearchSender.java

```diff
@@ -12,4 +12,6 @@ public interface KafkaSearchSender {
 
   void sendResourceDeleted(Resource resource);
 
+  void sendAuthorityCreated(Resource resource);
```

Comment:
I think it is better to rename the methods in this interface as follows to be consistent. If it is lot of changes for this ticket, I can create a small cleanup story for this change.

sendSingleResourceCreated  --> sendWorkCreated() 
sendMultipleResourceCreated --> sendMultipleWorksCreated()
sendResourceUpdated --> sendWorkUpdated()
sendResourceDeleted -> sendWorkDeleted()
sendAuthorityCreated -> Good no change.